.github/2.0/cookbooks/Executor.md

```diff
@@ -234,24 +256,25 @@ If you don't need some arguments, you can suppress it into `**kwargs`. For examp
 
 ```python
 @requests
-def foo(docs, **kwargs):
-    bar(docs)
+def foo_using_docs_arg(docs, **kwargs):
+    print(docs)
 
 
 @requests
-def foo(docs, parameters, **kwargs):
-    bar(docs)
-    bar(parameters)
+def foo_using_docs_parameters_arg(docs, parameters, **kwargs):
+    print(docs)
+    print(parameters)
 
 
 @requests
-def foo(**kwargs):
-    bar(kwargs['docs_matrix'])
+def foo_using_no_arg(**kwargs):
+    # the args are suppressed into kwargs
+    print(kwargs['docs_matrix'])
 ```
 
 ### Method Returns
 
-Method decorated with `@request` can return `Optional[DocumentSet]`. If not `None`, then the current `Request.docs` will
+Method decorated with `@request` can return `Optional[DocumentArray]`. If not `None`, then the current `Request.docs` will
```

Comment:
I would here maybe emphasize that the changes done in the `docs` affect the `Request.docs`. (That `Document` is just a memory view)